react/SelectBox/SelectBox.jsx

```diff
@@ -7,11 +7,15 @@ import { dodgerBlue, silver, coolGrey } from '../palette'
 import withBreakpoints from '../helpers/withBreakpoints'
 import classNames from 'classnames'
 
-const customStyles = {
-  container: base => ({
-    ...base,
-    maxWidth: '30rem'
-  }),
+// Heights compatible with Field component heights. We need to add an extra
+// pixel for each top and bottom border. Final heights will be two pixels less.
+const heights = {
+  tiny: '34px',
```

Comment:
why not `rem` values ?

Reply:
Let's try them !

Reply:
Do we agree on `2.125rem` ?

Reply:
Sizes are supposed to be:
- 32px
- 40px
- 48px
All those are Including the paddings and borders.

If the `box-sizing` is, as I think it is, `border-box` then those size in rem should be:
- 2rem
- 2.5rem
- 3rem

Reply:
Yep, put react-select seems to remove 2px for free, so I added them in the code. Want a demo ?

Reply:
yes please :/ that's an odd behaviour

Reply:
In your styleguide you set 2.125rem and it has a height of 2.125rem. I don't see where those 2px are removed :/